core/domain/classifier_services.py

```diff
@@ -359,9 +444,8 @@ def delete_classifier_training_job(job_id):
         classifier_training_job_model.delete()
 
 
-def get_classifier_from_exploration_attributes(exp_id, exp_version,
```

Comment:
Don't we need this function? How else would we get the classifier?

From what I understand. Now we are mapping <exp_id, version, state> to job_id but that should not change how we retrieve classifier. We still need same mapping as above to get to the classifier. May be I am missing something here?